wscript

```diff
@@ -339,7 +339,7 @@ def run_python_tests(ctx):
     os.system('cp -r src/python/essentia build/python/')
     os.system('cp build/src/python/_essentia*.so build/python/essentia')
 
-    ret = os.system('PYTHONPATH=build/python %s test/src/unittests/all_tests.py' % sys.executable)
+    ret = os.system('PYTHONPATH=build/python:$PYTHONPATH %s test/src/unittests/all_tests.py' % sys.executable)
```

Comment:
Maybe it'd be a good idea to use subprocess.run (python 3.5 and above), which takes a specific `env` argument, allowing us to run this without using a subshell. 
Double-check that this is valid if there is a space in `$PYTHONPATH`